### ISSUE TITLE
Failsafe Behavior in 3.0.1R4

### ISSUE THREAD

**Issue by bmizes** [MegaPirateNG/ardupilot-mpng#99]
I loaded 3.0.1R4 on my Crius 1.1 board and was testing the throttle failsafe behavior and found what appears to be a very disconcerting behavior.  Here's what I did:  With MP hooked up to my board via bluetooth and external power to the board only (main battery not plugged in so no power to the ESCs and motors), I armed the copter and it shows armed on the Flight Data screen.  I moved the throttle to mid-stick.  I then turned off my radio to simulate a loss of radio signal which then drops the throttle PWM below the 975 trigger point.  The flight data screen changes from ARMED to Failsafe, and the flight mode changes from Stabilize to Land as I had programmed.  All good up to this point.  About 1 second after the Failsafe display and the change of flight mode, the flight data screen shows DISARM.  My question is whether this behavior would occur if power was to the motor, at which point the copter would fall from the sky since it was disarmed, or whether the copter would stay armed if the motors were actually spinning.  If you don't know the answer to this, I guess I could have someone hold the copter while I spin it up and then turn off the radio, but I was hoping you had an answer to this.  Thanks.  Barry

**Comment by bmizes** [comment]
I just looked at the failsafe.pde file and it looks like the behavior I described above is intentional in that the code disarms the motors if failsafe persists for two seconds.  Why would we want to do this rather than have the copter go into either RTL or Land mode?  Disarming the motors in the air virtually guarantees a crash.  Why not let the RTL or Land function do its job and safely bring the copter home and/or land it?  Am I missing something here?  Thanks.  Barry
